ext_emconf.php

```diff
@@ -13,8 +13,8 @@
     'author_email' => 'manuel_selbach@yahoo.de',
     'constraints' => [
         'depends' => [
-            'typo3' => '11.5.0-11.5.99',
-            'static_info_tables' => '11.5.0-11.5.99',
+            'typo3' => '11.5.0-12.4.99',
+            'static_info_tables' => '11.5.0-12.4.99',
```

Comment:
I also submitted a PR for them, where I use 12.4. I decided that naming it by TYPO3 version is the best to show what ext is for what TYPO3 version. If they decide on a different tag, it can be changed here too.

Reply:
I'd like to wait for that, otherwise, we have to change it again. May I ask you to bring up an issue for that with the PR linked, to follow this?

Reply:
Yes, this is not for the immediate merge, this is just the code I did for a project that we upgrade. I was wrong about submitting a PR for the base ext, it was for `_de` and `_it`.

For the main ext, please, see their `master` branch. Their ext version for TYPO3 11.5 was also 11.5 (see their tags), so I assume they will have a 12.4 now.

Reply:
https://codeberg.org/sjbr/static-info-tables

Reply:
@manuelselbach @dmitryd 
https://extensions.typo3.org/extension/static_info_tables has a 12.x Tag. This can be merged.